workflow/rules/common.smk

```diff
@@ -360,7 +356,7 @@ def all_input(wildcards):
         )
     )
 
-    if is_activated(config["diffsplice"]):
+    if config["diffsplice"]["activate"]:
```

Comment:
Why this change?

Reply:
The `is_activated()` function was not working as intended (any more). Nowadays, the yaml seems to be parsing settings of `true` directly into a boolean, and that boolean was failing the string-based comparison. Also, this was the only place that function was still used. So instead of fixing the function, I removed it -- one layer less of abstraction.

Reply:
ok